actions/get-vault-secrets/action.yaml

```diff
@@ -62,6 +62,14 @@ runs:
         COMMON_SECRETS: ${{ inputs.common_secrets }}
         REPO: ${{ github.repository }}
 
+    - id: get-github-jwt-token
+      uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+      with:
+        script: |
+          const jwt = await core.getIDToken("vault-github-actions-grafana");
```

Comment:
(not required from my perspective) - I would put the instance in here. It just means you can't use a `dev` ID token with the `ops` proxy and the other way around. That's what "audience" means to me: who is this token for? And we can be quite specific in this case 🙂